ejb3/src/main/java/org/jboss/as/ejb3/component/EJBComponent.java

```diff
@@ -274,7 +275,7 @@ public Principal getCallerPrincipal() {
         }
 
         // TODO Should this return a Principal when security is not activated.
```

Comment:
Just minor, but we can remove this TODO comment now. :)

Reply:
Looks like this got merged with the TODO still in place :-P

Reply:
@rhusar I submitted PR that removes the TODO comment https://github.com/wildfly/wildfly/pull/15238 :tada: